multimedia/tvheadend/Config.in

```diff
@@ -43,6 +43,12 @@ choice
       Use more advanced Perl-Compatible Regular Expressions, provided by libpcre2.
 endchoice
 
+config TVHEADEND_URIPARSER
+  bool "Use liburiparser"
+  depends on PACKAGE_tvheadend
+  select PACKAGE_liburiparser
+  default n
```

Comment:
Now, this is the only config, which does not have description.

Reply:
fixed.